### ISSUE TITLE
Empty condition in WAF-Tags

### ISSUE THREAD

**Issue by 8-bit-fox** [Myra-Security-GmbH/terraform-provider-myrasec#175]
If I add a "match-all"-WAF-tag-rule (without) condition
and import it in a Terraform state
I get the resource configuration
```
      {
          actions        = [
              {
                  available_phases    = 1
                  custom_key          = ""
                  force_custom_values = false
                  name                = "Block"
                  type                = "block"
                  value               = ""
                },
            ]
          conditions     = []
          description    = "Matches everything else"
          direction      = "in"
          enabled        = true
          name           = "Default Block"
          process_next   = false
          sort           = 1
      },
```
But when I try to create a corresponding ressource, Terraform fails complaining that the condition block is invalid.

**Comment by techdecline** [comment]
I am having the same problem.

**Comment by smashi** [comment]
Hi, 
we released a new version v1.68.2 with a fix for this.
